nucleus/admin/cli/src/main/java/com/sun/enterprise/admin/cli/CLICommand.java

```diff
@@ -1440,12 +1440,24 @@ protected void buildTerminal() {
     
     protected void buildLineReader() {
         if (lineReader == null) {
-            lineReader = LineReaderBuilder.builder()
+            lineReader = newLineReaderBuilder()
                     .terminal(terminal)
                     .build();
         }
     }
     
+    protected LineReaderBuilder newLineReaderBuilder() {
+    	// In community this should be disabled by default
+    	boolean disabled = true;
+    	if(System.getenv("DISABLE_EVENT_EXPANSION") != null) {
```

Comment:
```
/**
 * The environment variables for CLI commands.  An instance of this class
 * is passed to each command to give it access to environment variables.
 * Command implementations should access environment information from
 * this class rather than using System.getenv.  In multimode, the export
 * command may change environment variables in the instance of this class
 * that is shared by all commands.
 *
 * @author Bill Shannon
 */
public final class Environment {
```